src/stores/analyticsContext.tsx

```diff
@@ -50,6 +74,24 @@ const useAnalytics = ({
     windowWithAnalytics._paq.push(args)
   }
 
+  // Shortcut for push(['trackEvent', ...])
```

Comment:
this is so great, I love how readable it makes the code wherever it's being used